### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: src/Accordion/Accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    const result = await fetch(`https://jsonplaceholder.typicode.com/posts`);
+    const resultJson = await result.json();
+    setData(resultJson);
+  }
+  return (
+    <>
+      <h1 className="main-title">How can we help you?</h1>
+      {data.map(({ title, body, id }) => {
+        return (
+          <Fragment key={id}>
```
Comment (takrami) [inline src/Accordion/Accordion.jsx]: fragment is extra here. you can put key on AccordionItem. Please send a voice to explain when we use fragment.

File: src/AccordionItem/AccordionItem.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+function AccordionItem({ title, body, id }) {
+  const [isActive, setIsActive] = useState(false);
+
+  function handleActivity() {
+    setIsActive(!isActive);
+  }
+
+  return (
+    <Fragment key={id}>
```
Comment (takrami) [inline src/AccordionItem/AccordionItem.jsx]: fragment can be removed.

File: src/Accordion/Accordion.jsx (hunk 1 above)
Comment (hedita) [reply src/Accordion/Accordion.jsx]: Done.

File: src/AccordionItem/AccordionItem.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <div className="accordion">
+        <div className="accordion-title" onClick={handleActivity}>
+          {title}
+          <div>{isActive ? "▴" : "▼"}</div>
+        </div>
+        {isActive && <div className="accordion-content">{body}</div>}
+        <hr />
```
Comment (takrami) [inline src/AccordionItem/AccordionItem.jsx]: please remove this and handle the border with css.

File: src/AccordionItem/AccordionItem.jsx (hunk 2 above)
Comment (hedita) [reply src/AccordionItem/AccordionItem.jsx]: Done.

File: src/AccordionItem/AccordionItem.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  const [isActive, setIsActive] = useState(false);
+
+  function handleActivity() {
+    setIsActive(!isActive);
+  }
+
+  return (
+    <Fragment key={id}>
+      <div className="accordion">
+        <div className="accordion-title" onClick={handleActivity}>
```
Comment (takrami) [inline src/AccordionItem/AccordionItem.jsx]: can't we use a semantic tag here? maybe h2?

File: src/AccordionItem/AccordionItem.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+          {title}
+          <div>{isActive ? "▴" : "▼"}</div>
+        </div>
+        {isActive && <div className="accordion-content">{body}</div>}
```
Comment (takrami) [inline src/AccordionItem/AccordionItem.jsx]: can't we use a semantic tag here? maybe p?

File: src/AccordionItem/AccordionItem.jsx (hunk 3 above)
Comment (hedita) [reply src/AccordionItem/AccordionItem.jsx]: Done.

File: src/AccordionItem/AccordionItem.jsx (hunk 4 above)
Comment (hedita) [reply src/AccordionItem/AccordionItem.jsx]: Done.

File: src/AccordionItem/AccordionItem.jsx (hunk 5 above)
Comment (hedita) [reply src/AccordionItem/AccordionItem.jsx]: Done.

**Review by takrami — APPROVED** [review]

**Pull request merged**